src/backend/TrafficCourts/Citizen.Service/Services/Tickets/Search/Mock/test-ticket-data.csv

```diff
@@ -1,64 +1,64 @@
-ST046347521,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-0812:21,167,167,Disobey red light at intersection,,2022-05-08,
-ST046332581,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T11:14,167,167,Disobey red light at intersection,,2022-05-08,
-ST046654221,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T10:58,167,167,Disobey red light at intersection,,2022-05-08,
-ST046255531,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T16:15,167,167,Disobey red light at intersection,,2022-05-08,
-ST046125881,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T14:21,167,167,Disobey red light at intersection,,2022-05-08,
-ST046395221,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T19:11,167,167,Disobey red light at intersection,,2022-05-08,
-ST046334421,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T12:18,167,167,Disobey red light at intersection,,2022-05-08,
-SG008537641,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T12:49,196,196,Speed in municipality if the speed limit is exceeded by 21 km/hr to 40 km/hr,,2022-05-08,
-SG008552231,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T08:06,196,196,Speed in municipality if the speed limit is exceeded by 21 km/hr to 40 km/hr,,2022-05-08,
-SG008562631,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T15:58,368,368,Speed in municipality if the speed limit is exceeded by 41 km/hr to 60 km/hr,,2022-05-08,
-SG008562971,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T15:18,368,368,Speed in municipality if the speed limit is exceeded by 41 km/hr to 60 km/hr,,2022-05-08,
-SG008551151,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T19:19,196,196,Speed in municipality if the speed limit is exceeded by 21 km/hr to 40 km/hr,,2022-05-08,
-SG008522181,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T08:26,368,368,Speed in municipality if the speed limit is exceeded by 41 km/hr to 60 km/hr,,2022-05-08,
-SG008561271,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-03-23T13:03,368,368,Speed in municipality if the speed limit is exceeded by 41 km/hr to 60 km/hr,,2022-03-23,
-SG008561881,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-03-24T08:57,196,196,Speed in municipality if the speed limit is exceeded by 21 km/hr to 40 km/hr,,2022-03-24,
-SG008551191,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-03-25T22:03,196,196,Speed in municipality if the speed limit is exceeded by 21 km/hr to 40 km/hr,,2022-03-25,
-SG008537611,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-03-26T21:21,196,196,Speed in municipality if the speed limit is exceeded by 21 km/hr to 40 km/hr,,2022-03-26,
+﻿invoice_number,pbc_ref_number,party_number,party_name,account_number,site_number,cust_trx_type,term_due_date,total,amount_due,attribute1,attribute2,attribute3,attribute4,act,section
+EA020004601,n/a,n/a,n/a,n/a,n/a,Traffic Violation Ticket,2022-05-08T09:54,139,139,Speed against area sign,,2022-05-08,n/a,,
```

Comment:
I thought it would be good to do that. I it would be easy to set them all the same value.  Perhaps we can get someone else to do that like the BA or the tester? I dont want to be on the hook for test data.

Reply:
Sounds good to have the BA do this.

Reply:
I have requested either the BA / Tester make some adjustments. Will merge once these are done.